lib/ansible/modules/stat.py

```diff
@@ -346,6 +354,12 @@
             type: list
             sample: [ immutable, extent ]
             version_added: 2.3
+        selinux_context:
+            description: The SELinux context of a file/directory
```

Comment:
`of a path`, no need to repeat file/directory everywhere ... and some paths are neither.